package.json

```diff
@@ -69,10 +69,10 @@
     "eslint-config-airbnb": "1.0.2",
     "eslint-plugin-react": "^4.1.0",
     "mocha": "^2.2.5",
-    "redux": "^3.4.0",
+    "redux": "~4.0.0",
     "rimraf": "^2.5.2",
-    "typescript": "^1.8.10",
-    "typescript-definition-tester": "0.0.4",
+    "typescript": "~2.6.2",
```

Comment:
Eeeehhhh.... not sure here; typescript does use the 2nd dot as the "major" version, but probably still better to use `^`.

Reply:
Would be nice to make it so there's no 3rd parameter if `E` is `never` or `undefined`.